lib/opam.mli

```diff
@@ -22,11 +22,12 @@ val prepare :
   remote_repo:string ->
   opam_repo:string * string ->
   version:string ->
-  string list ->
+  main_pkg:string ->
+  all_pkgs:string list ->
   (string, R.msg) result
-(** [prepare ~local_repo ~version pkgs] adds the packages [pkg.version] to a new
-    branch in the local opam repository [local_repo], using the commit message
-    [msg] (if any). Return the new branch. *)
+(** [prepare ~local_repo ~version ~main_pkg ~all_pkgs] adds the packages
+    [pkg.version] to a new branch in the local opam repository [local_repo],
+    using the commit message [msg] (if any). Return the new branch. *)
```

Comment:
I don't quite understand this documentation (although, to be honest, I also didn't understand the old one). What is `[pkg.version]`? And what does the whole sentence mean? Does it mean that the packages from `[all_pkgs]` are added to a new branch named `release-[main_package]-[version]` in `[local_repo]`? And, if so, doesn't the function do a bit more, such as also committing and pushing that change?

Reply:
Indeed! I will update the documentation